src/todomanager/popup.tsx

```diff
@@ -42,36 +46,101 @@ function Popup() {
     );
   };
 
+  const deleteTodo = (id: string) => {
+    setTodos(todos.filter(todo => todo.id !== id));
+  };
+
+  const handleEditTodo = (e: React.ChangeEvent<HTMLInputElement>) => {
+    if (editTodo) {
+      setEditTodo({ ...editTodo, content: e.target.value });
+    }
+  };
+  const startEditTodo = (todo: Todo) => {
+    setEditTodo({ id: todo.id, content: todo.content });
+  };
```

Comment:
```suggestion
  const startEditTodo = (e: React.MouseEvent<HTMLButtonElement>) => {
      const id = e.currentTarget.value;
      const todoToEdit = todos.find(todo => todo.id === id);
      if (todoToEdit) {
        setEditTodo({ id: todoToEdit.id, content: todoToEdit.content });
      }
    };
```
Nit: 함수를 다음과 같이 선언 한 후에 `button` 태그에 id값을 할당하여 사용하는 방법도 좋을 것 같습니다!

```
  <button
    type="button"
    value={todo.id}
    onClick={startEditTodo}
    className="text-blue-500">
    수정
  </button>
```

Reply:
현재는 버튼에는 value 속성을 부여할 필요가 없을 것 같습니다 😸